Resources/Prototypes/Entities/Objects/Weapons/Melee/telescopic_baton.yml

```diff
@@ -0,0 +1,71 @@
+﻿- type: entity
+  id: TelescopicBaton
+  parent: BaseItem
+  name: telescopic baton
+  description: A compact and harmless personal defense weapon. Sturdy enough to knock the feet out from under attackers.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/telebaton.rsi
+    layers:
+    - state: icon
+      map: [ "enum.TelescopicBatonVisuals.Layer" ]
+  - type: Item
+    size: Small
+  - type: ItemToggle
+    soundActivate:
+      path: /Audio/Weapons/telescopicon.ogg
+      params:
+        volume: -2
+    soundDeactivate:
+      path: /Audio/Weapons/telescopicoff.ogg
+      params:
+        volume: -2
+  - type: ItemToggleDisarmMalus
+    activatedDisarmMalus: 0.6
+  - type: ItemToggleMeleeWeapon
+    activatedDamage:
+      types:
+        Blunt: 12
```

Comment:
Sad i enjoyed my Baton doing no damage but only knock down so its gave them time to run.

Personnaly il prefer it to be only knock down or minimal damage near 1-5, making them avoid using in combat and more as a defense tool only.

Reply:
Have you ever been... beaten by this thing?
IT HURTS. IT HURTS ALOT.

Reply:
I haven been and did beat peoples A LOT WITH IT.

But again that was on SS13/Paradise Station and damage there i think was near complete ZERO